packages/create-strapi-app/create-strapi-app.js

```diff
@@ -2,7 +2,9 @@
 
 const commander = require('commander');
 const packageJson = require('./package.json');
-const generateNewApp = require('strapi-generate-new');
+// TODO: restore this line instead:
+// const generateNewApp = require('strapi-generate-new');
+const generateNewApp = require('../strapi-generate-new');
```

Comment:
This will be reverted before merging, it's just to test locally

Reply:
I reverted it back the original to avoid messing up the PR reviews. Change this line to a relative require if you want to try templates locally